### PR TITLE
LPS-180871 create script to migrate products marketplace

### PR DESCRIPTION
**Pull request by thiagooliveiraaa**

### CODE REVIEW

**Comment by thiagooliveiraaa** [comment]
@nNilton

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-script/src/main/java/com/liferay/marketplace/Main.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+			String liferayMarketplaceOAuthClientId,
+			String liferayMarketplaceOAuthClientSecret, URL liferayURL)
+		throws Exception {
+
+		_liferayURL = liferayURL;
+
+		_liferayOAuthClientId = liferayMarketplaceOAuthClientId;
+		_liferayOAuthClientSecret = liferayMarketplaceOAuthClientSecret;
+
+		System.out.println("Liferay URL: " + _liferayURL);
```
Comment (nNilton) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-script/src/main/java/com/liferay/marketplace/Main.java:79]: switch to log.info

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-script/src/main/java/com/liferay/marketplace/Main.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+	public Main(
+			String liferayMarketplaceOAuthClientId,
+			String liferayMarketplaceOAuthClientSecret, URL liferayURL)
+		throws Exception {
+
+		_liferayURL = liferayURL;
+
+		_liferayOAuthClientId = liferayMarketplaceOAuthClientId;
+		_liferayOAuthClientSecret = liferayMarketplaceOAuthClientSecret;
```
Comment (nNilton) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-script/src/main/java/com/liferay/marketplace/Main.java:77]: sort

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-script/src/main/java/com/liferay/marketplace/Main.java [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+				"dependencies/application.properties");
+			Properties appProps = new Properties();
+
+			appProps.load(inputStream);
+
+			Main main = new Main(
+				appProps.getProperty("LIFERAY_MARKETPLACE_OAUTH_CLIENT_ID"),
+				appProps.getProperty("LIFERAY_MARKETPLACE_OAUTH_CLIENT_SECRET"),
+				new URL(
+					appProps.getProperty("LIFERAY_MARKETPLACE_LIFERAY_URL")));
```
Comment (nNilton) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-script/src/main/java/com/liferay/marketplace/Main.java:62]: let's try with a void method

**Review by nNilton — CHANGES_REQUESTED** [review]

**Pull request closed without merging**